cmd/capdctl/main.go

```diff
@@ -242,37 +248,43 @@ func machineYAML(opts *machineOptions) string {
 }
 
 func makeManagementCluster(clusterName string) {
-	kind := execer.NewClient("kind")
-	// start kind with docker mount
-	kindConfig, err := kindConfigFile()
+	fmt.Println("Creating a brand new cluster")
+	elb, err := actions.SetUpLoadBalancer(clusterName)
 	if err != nil {
 		panic(err)
```

Comment:
out of scope of this PR, but the `panic()` pattern here is not recommended.
errors should bubble up to an exit(1).

Reply:
...or `klog.Fatal`

Reply:
Right now i'm keeping panic for consistency. I opened https://github.com/kubernetes-sigs/cluster-api-provider-docker/issues/8 to get a new contributor involved but will probably timeout on the issue soon since this can't live for much longer